enterprise/server/configure.md

```diff
@@ -108,14 +104,14 @@ ephox {
 
 {{site.requires_jsscwar_230v}}
 
-Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character.
+Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character. When using regular expressions, the `scheme` and `port` is not excluded from the matching process.
```

Comment:
```suggestion
Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character. When using regular expressions, the `scheme` and `port` are not excluded from the matching process.
```

Reply:
Entire sentence was deleted, because it was wrong.